include/git2/worktree.h

```diff
@@ -137,23 +161,46 @@ typedef enum {
 	GIT_WORKTREE_PRUNE_WORKING_TREE = 1u << 2,
 } git_worktree_prune_t;
 
+typedef struct git_worktree_prune_options {
+	unsigned int version;
+
+	char prune_valid; /**< prune even if working tree is valid */
+	char prune_locked; /**< prune even if working tree is locked */
+	char delete_worktree; /**< delete complete working directory */
```

Comment:
Okay, I'll change it back to use flags again, but will keep them inside the struct.